common/utils/src/main/resources/error/error-classes.json

```diff
@@ -5271,7 +5271,7 @@
   },
   "_LEGACY_ERROR_TEMP_1148" : {
```

Comment:
Could you add a sub-class to `UNSUPPORTED_INSERT` with a name like `MULTI_PATH`.

Reply:
Okay, I'll update it later.